includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -38,6 +40,31 @@ public function track_product_updated( $product_id, $post ) {
 		);
 
 		WC_Tracks::record_event( 'product_edit', $properties );
+
+		$product_factory = new WC_Product_Factory();
+		$product = $product_factory->get_product( $product_id );
+		$update_properties = array(
+			'product_id'            => $product_id,
+			'product_type'          => $product->get_type(),
+			'is_virtual'            => $product->get_virtual(),
+			'is_downloadable'       => $product->get_downloadable(),
+			'manage_stock'          => 0 == $product->get_manage_stock() ? 'N' : 'Y',
+		);
+
+		WC_Tracks::record_event( 'product_update', $update_properties );
```

Comment:
I've got some JS that only triggers when clicking update on the product page, but how would I get the product details to fill in `properties`?

```
		wc_enqueue_js(
			"
			if ( jQuery( 'h1.wp-heading-inline' ).text().trim() === 'Edit product') {
				jQuery( '#publish' ).click( function() {
					const properties = {
						//???
					};
					window.wcTracks.recordEvent( 'product_update', properties );
				} );
			}
			"
		);
```

Reply:
Would something like this work?

```php
wc_enqueue_js(
    "
        if ( jQuery( 'h1.wp-heading-inline' ).text().trim() === 'Edit product') {
            jQuery( '#publish' ).click( function() {
                const properties = {
                    url: '<?php echo home_url(); ?>',
                };
                window.wcTracks.recordEvent( 'product_update', properties );
            } );
        }
    "
);

```

Reply:
I guess I could only insert the JS when the product details are available

Reply:
I ended up pulling the product details using jQuery

![](https://media.giphy.com/media/14kdiJUblbWBXy/giphy.gif)

Reply:
We are only able now to track stores in English because of `jQuery( 'h1.wp-heading-inline' ).text().trim() === 'Edit product'`, this need to be improved.